OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListFragment.java

```diff
@@ -257,7 +258,8 @@ public void onClick(View v) {
         footer.courseNotListedTv.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                environment.getRouter().showWebViewActivity((getActivity()), getString(R.string.course_not_listed_file_name), null);
+                AlertDialogFragment.newInstance(null, getString(R.string.cant_find_course_prompt), null)
```

Comment:
Shouldn't we use `showAlertDialog` function of `BaseFragmentActivity` class here like this ?
`((BaseFragmentActivity) getActivity()).showAlertDialog(null, getString(R.string.cant_find_course_prompt), null);`